packages/integration-tests/src/integration/create-sno/0-create-cluster.spec.js

```diff
@@ -22,11 +20,11 @@ describe(`Assisted Installer SNO Cluster Installation`, () => {
       clusterListPage.getCreateNewClusterButton().should('be.visible');
       clusterListPage.getCreateNewClusterButton().click();
 
-      cy.location('pathname').should('eq', NEW_CLUSTER_URL);
+      cy.location('pathname').should('eq', '/clusters/~new');
```

Comment:
If this URL (`/clusters/~new/`) changes in the future we'll need to replace it all over the place.
Since it can be regarded as a property of the clustersListPage, why don't you create a property such as `clustersListPage.url` holding that value?

Reply:
Hi, I did use this kind of value. I didn't write the code for this file

> > I also want to add deletion of tang server, but for that, I need to add data-testid in the lib. You said we need to discuss it so tell me when you can.
> 
> Regarding the `data-testid` attr, take a look at some of the current implementations in the lib, PTAL at the type `WithTestId`.
> 
> BTW, the Tang servers' delete button looks a bit weird.

What do you mean by weird?